### PR TITLE
Corrections Intro.ipynb and Intro_es.ipynb

### PR DESCRIPTION
**Pull request by ChavezUriel**
@alxogm Can you take a look to these corrections?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,11 +33,11 @@ Everything here is written in [Jupyter notebooks](https://jupyter.org/), which m
 If you get stuck with a python question, be like the pros and try www.google.com!  :thought_balloon: If you have a science question,
 leave us a [message](https://github.com/michaelJwilson/DESI-HighSchool/issues/new/) and we'll be happy to help answer them.  But remember, all staff and students at DESI High abide by the [Code of Conduct](https://github.com/michaelJwilson/DESI-HighSchool/blob/master/CODE_OF_CONDUCT.md) :scroll:.  
 
-When you're done, don't forget to get some extra credit by leaving us with what you [liked](https://forms.gle/LGKMVamrtS5StSv56) :star:.  You can show even more support by [donating](berkeleylabfoundation.org/support) a few :dollar: to our support programs for the underprivileged, but make sure to convince your parents first!
+When you're done, don't forget to get some extra credit by leaving us with what you [liked](https://forms.gle/LGKMVamrtS5StSv56) :star:.  You can show even more support by [donating](https://www.berkeleylabfoundation.org/support) a few :dollar: to our support programs for the underprivileged, but make sure to convince your parents first!
```
Comment (alxogm) [inline README.md:36]: @michaelJwilson I'm not sure if the links are broken on purpose? or if before they were working well as they were... Do you agree with this being fixed?

**Review by alxogm — APPROVED** [review]
Apart from my question to Mike, I think this can be merged. 

(I'm not sure why I can't tag Uriel, but thanks to him for this corrections)

File: README.md (hunk 1 above)
Comment (michaelJwilson) [reply README.md:36]: yeh, this is fine.  Not sure I'll wild about pointing there, but it shouldn't be an issue.

**Comment by michaelJwilson** [comment]
@ChavezUriel Can you perhaps change https://github.com/ChavezUriel/desihigh/blob/master/Espanol/CODE_OF_CONDUCT_es.md

to point to the original https://github.com/michaelJwilson/...  Less likely to run into problems. 
Thanks for the fixes!

**Review by alxogm — APPROVED** [review]

**Comment by michaelJwilson** [comment]
@ChavezUriel @alxogm Apologies for the annoying timing, but I've recently updated the English README and you _might_ want to make similar changes in this PR (entirely) your call.

**Comment by alxogm** [comment]
> @ChavezUriel @alxogm Apologies for the annoying timing, but I've recently updated the English README and you _might_ want to make similar changes in this PR (entirely) your call.

I have gone through the changes to incorporate them (thanks for the explicit mention :D), and resolved a conflict that was present in the English version. I think is ready to merge if you approve @michaelJwilson .

**Comment by michaelJwilson** [comment]
Couldn't track the conflict, but I trust you :) thanks!

**Pull request merged**